hw/net/rocker/rocker_p4_l2l3_enums.h

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2013-present Barefoot Networks, Inc. 
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Its a generated file.. let me try to fix the tool that generates it.

On Sun, Apr 19, 2015 at 9:59 PM, Scott Feldman <notifications@github.com>
wrote:

> In hw/net/rocker/rocker_p4_l2l3_enums.h
> <https://github.com/pbhide/qemu-rocker/pull/1#discussion_r28664188>:
>
> > @@ -0,0 +1,41 @@
> > +/*
> > +Copyright 2013-present Barefoot Networks, Inc.
> > +
> > +Licensed under the Apache License, Version 2.0 (the "License");
>
> Why is this one under a diff license?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pbhide/qemu-rocker/pull/1/files#r28664188>.
>